bundles/http/Citrix/2ec16a1b-9423-45a6-8fd8-b0c42c916c0f/e40510b3-a760-43b4-ab16-56572dc6303b.js

```diff
@@ -0,0 +1,887 @@
+//Loaded moment-timezone library from sdk provided libraries 
+const moment = library.load('moment-timezone');
+
+
+// loading the users data(Done)
+async function syncUsers(dataStore, client, context) {
+    const page_size = 100
+    let next_page_token = ""
+    let userids = [];
+    do {
+        let request = await client.fetch(`/users?next_page_token=${next_page_token}&page_size=${page_size}`);
+        if (!(await request.ok)) {
+            throw new Error(await request.json());
+        }
+        let response = await request.json();
+        next_page_token = response['next_page_token'] ?? "";
+        let data = response['users']

```

Comment:
```suggestion
        const response = await client.fetch(`/users?next_page_token=${next_page_token}&page_size=${page_size}`);
        const json = await response.json();
        if (!response.ok)) {
            throw new Error(json);
        }
        
        next_page_token = json.next_page_token ?? "";
        const data = json.users;

```

Reply:
- `await` in `await request.ok` is not needed
- `const json = await response.json();` is better to do on one place.